### DISCUSSION TITLE
Move MP GraphQL from standalone to platform

### DISCUSSION THREAD

**Discussion by t1** [smallrye/smallrye-graphql#1129] [General]
In [this](https://docs.google.com/document/d/1LPIclkafszH9_l75iBYXzPwHYDQnDUh_moPrtxAGX4w/edit?pli=1) document, Emily Jiang writes: "I would like to see MicroProfile GraphQL move from standalone to platform."

We should work on preparing that step!

**Comment by phillip-kruger** [comment]
I guess that is more a MicroProfile effort than a SmallRye one ?

**Reply by t1** [reply]
🤪
